public/main.js

```diff
@@ -0,0 +1,54 @@
+// setting the map's view to Burlington
+let myMap = L.map("map").setView([44.4798, -73.2143], 14)
+
+// adding tile layet
+L.tileLayer('https://{s}.tile.opentopomap.org/{z}/{x}/{y}.png', {
+  maxZoom: 17,
+  attribution: 'Map data: &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, <a href="http://viewfinderpanoramas.org">SRTM</a> | Map style: &copy; <a href="https://opentopomap.org">OpenTopoMap</a> (<a href="https://creativecommons.org/licenses/by-sa/3.0/">CC-BY-SA</a>)'
+}).addTo(myMap)
+
+// adding marker to downtown Burlington on map
+let marker = L.marker([44.4798, -73.2143])
+let downtown = "Downtown Burlington"
+marker.bindPopup(`${downtown}`)
+marker.addTo(myMap)
+
+// adding an mouseover event listener for pop up over marker
+marker.addEventListener('mouseover', () => {
+  marker.openPopup()
+})
+
+// adding highlighting circle to downtown Burlington marker on map
+L.circle([44.4798, -73.2143], {
+  color: 'red',
+  fillColor: '#f03',
+  fillOpacity: 0.5,
+  radius: 500
+}).addTo(myMap);
+
+
+// function to place markers at each restaurant's location on the map
+function placeMarker(coords, name, id) {
+  let mapMarkers = L.marker(coords)
+  mapMarkers.addTo(myMap).bindPopup(`<a href='/restaurant?q=${id}'>${name}</a>`)
+}
+
+// mapMarkers.addEventListener('mouseover', () => {
+//   mapMarkers.openPopup()
+// })
```

Comment:
Having unused and duplicated code through your project leads to having a lot of bloat making it harder to navigate and read